packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-development.md

```diff
@@ -38,11 +38,11 @@ Note that ARIA can only affect the semantics of an element; it has no effect on
 
 #### Form fields
 
-Use [explicit linking](https://www.w3.org/TR/WCAG20-TECHS/H44.html) between `label` and form input elements (such as `input`, `textarea`, or `select`) when both are present. [This "H44" guidelines](//www.w3.org/TR/WCAG20-TECHS/H44.html) provides an accessible name to screen readers and makes the label clickable to increase the overall clickable area of the form element.
+[Use label elements to associate text labels with form controls](https://www.w3.org/WAI/WCAG22/Techniques/html/H44) (such as `input`, `textarea`, or `select`) when both are present. This technique provides an accessible name to screen readers and makes the label clickable to increase the overall clickable area of the form element.
 
-When a `label` element cannot accompany a form input element, provide one using using `aria-label` or `aria-labelledby`, as outline by the ["ARIA14" guidelines](//www.w3.org/TR/WCAG20-TECHS/ARIA14.html). 
+When a `label` element cannot accompany a form input element, provide one using using `aria-label` or `aria-labelledby`, as outline by the [WCAG technique ARIA14](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA14). 
 
-In a single-field form, the submit button label can serve as the field label for sighted users, [following the "G167" guidelines](//www.w3.org/TR/WCAG20-TECHS/general.html#G167). For assistive devices, use `aria-labelledby`.
+In a single-field form, the submit button label can serve as the field label for sighted users, followwing the [WCAG technique G167](https://www.w3.org/WAI/WCAG22/Techniques/general/G167). For assistive devices, use `aria-labelledby`.
```

Comment:
```suggestion
In a single-field form, the submit button label can serve as the field label for sighted users, following the [WCAG technique G167](https://www.w3.org/WAI/WCAG22/Techniques/general/G167). For assistive devices, use `aria-labelledby`.
```

Reply:
My w key has been my bane for a while now because of this 😭